frontend/src/main/webapp/pagevisit.js

```diff
@@ -0,0 +1,27 @@
+function pageData() {
```

Comment:
Same comments apply as in the trip list page (#20). Specifically, this function could use a clearer name and it would be better if you could use a simulated `fetchVisit` call.